api/metrics-api-jaxrs/README.adoc

```diff
@@ -170,7 +170,7 @@ GET /hawkular-metrics/{tenantId}/metrics
 *Example Request*
 
 ----
-curl http://localhost:8080/hawkular-metrics/{tenantId}/metrics?type=num
+curl http://localhost:8080/hawkular/metrics/{tenantId}/metrics?type=guage
```

Comment:
Weren't we supposed to leave these behind and use consistent naming everywhere? avail vs. availability and ?type parameter instead of /resource?